kernel/proc_list.c

```diff
@@ -0,0 +1,67 @@
+#include "types.h"
```

Comment:
Looks like code repetition from list.c, but maybe it is ok

Reply:
Yes but with proc as value for each node